[plugin-azure-key-vault] Add ability to read property values from Azure KeyVault (#5182)
vividus-framework/vividus: docs/modules/configuration/pages/secrets-management.adoc

db.connection.test.username=HASHI_CORP_VAULT(secret/vividus/test/username)
db.connection.test.password=HASHI_CORP_VAULT(secret/vividus/test/password)
----

=== xref:plugins:plugin-azure-resource-manager.adoc#_secrets_management_tools[Azure Key Vault]

> ==== Configuration
> 
> Supported getting property values from Azure Key Vault.
> Please see xref:plugins:plugin-azure-resource-manager.adoc#_configuration[Configuration section] to get more
> information on how configure required plugin.

Reviewer: ```suggestion
Check the xref:plugins:plugin-azure-resource-manager.adoc#_configuration[corresponding section] in Azure Resource Manager Plugin.
```

also make sure link points to `Secrets Management Tools` section

Author: added link on `_secrets_management_tools` in plugin-azure-resource-manager.adoc

db.connection.test.username=HASHI_CORP_VAULT(secret/vividus/test/username)
db.connection.test.password=HASHI_CORP_VAULT(secret/vividus/test/password)
----

> === xref:ROOT:plugin-azure-resource-manager.adoc#_secrets_management_tools[Azure Key Vault]

Reviewer: add basic description, like "AKV is supported, but it requires plugin, check details here"
